tests/ci/cdk/cdk.context.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
I know we talked about not updating this file. But will the cdk run work with it being deleted, or will it be automatically generated again?

Reply:
i should have called this out in the README. i tested that by deleting this file, then running the `update-ci` action, which worked. it even re-created this file, which should subsequently be `.gitignore`d after this is merged.

Reply:
That's actually pretty neat, looks like I should apply to AWS-LC as well.